sa/_db-next/migrations/20170731115209_AddOrders.sql

```diff
@@ -5,12 +5,13 @@ CREATE TABLE orders (
        id BIGINT(20) NOT NULL AUTO_INCREMENT,
        registrationID BIGINT(20) NOT NULL,
        expires DATETIME NOT NULL,
-       csr MEDIUMBLOB NOT NULL,
```

Comment:
Because this is going to get squashed and has really nothing to do with the rest of this change I'd rather this be done in a separate PR.

Reply:
Ok, restored in 7c8f4c4

Reply:
Oops. Missed part of the restoration. Fixed in f4ebe4f